spyder/app/mainwindow.py

```diff
@@ -1396,6 +1397,10 @@ def post_visible_setup(self):
         if not running_under_pytest():
             self.report_missing_dependencies()
 
+        # Show tour dialog
+        self.tour_dialog = tour.OpenTourDialog(self, lambda: self.show_tour(0))

```

Comment:
What might be best here is to declare a module-level constant `DEFAULT_TOUR_INDEX` and use that both here and in PR #13717 but we'd need to merge that first, rebase and then make the change.